deploy/crds/api.astarte-platform.org_v1alpha1_astarte_cr.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: api.astarte-platform.org/v1alpha1
+kind: Astarte
+metadata:
+  name: example-astarte
```

Comment:
or should we have an examples directory?

Reply:
This is directly retrocompatible to what we used to have in the previous operator. If we want to make this move it might be ok, but it's out of the scope of this specific PR